### PR TITLE
[ECO-2015] Update rewards popup and confetti

### PR DESCRIPTION
**Pull request by CRBl69**
# Description

This PR updates the confetti and reward popup to match Figma.

image

It is not shown here but the confetti only appear inside the box just like in Figma.

# Checklist

- [x] Did you check all checkboxes from the linked Linear task?

### CODE REVIEW

**Comment by CRBl69** [comment]
- [ ] Update spacing

File: src/typescript/frontend/tailwind.config.js [hunk 1]
```diff
@@ -32,8 +32,7 @@ module.exports = {
       boxShadow: {
         pretty:
           "-0.5px 0.5px 0.3px rgba(0,0,0,0.04), -1.1px 1.1px 0.7px rgba(0,0,0,0.055), -1.8px 1.8px 1.2px rgba(0,0,0,0.064), -2.7px 2.7px 1.8px rgba(0,0,0,0.071), -3.9px 3.9px 2.6px rgba(0,0,0,0.077), -5.5px 5.5px 3.7px rgba(0,0,0,0.083), -7.8px 7.8px 5.3px rgba(0,0,0,0.089)",
-        econia:
-          `-1px 1px 7px 5px rgba(8, 108, 217, .12) !important`
+        econia: `-1px 1px 7px 5px rgba(8, 108, 217, .12) !important`,
```
Comment (xbtmatt) [inline src/typescript/frontend/tailwind.config.js:35]:
```suggestion
        econia: "-1px 1px 7px 5px rgba(8, 108, 217, .12) !important",
```

File: src/typescript/frontend/src/components/pages/emojicoin/components/trade-emojicoin/SwapButton.tsx [hunk 2]
```diff
@@ -52,13 +52,21 @@ export const SwapButton = ({
...
-          closeOnClick: false,
-        });
+        toast.dark(
+          <>
+            <RewardsAnimation controls={controls} />
+            <CongratulationsToast transactionHash={res.response.hash} />
+          </>,
+          {
+            pauseOnFocusLoss: false,
+            autoClose: 7000,
```
Comment (xbtmatt) [inline src/typescript/frontend/src/components/pages/emojicoin/components/trade-emojicoin/SwapButton.tsx:62]: Just a reminder that we were talking about increasing this to about double, so maybe 15000 here

Comment (CRBl69) [reply src/typescript/frontend/src/components/pages/emojicoin/components/trade-emojicoin/SwapButton.tsx:62]: Thank you. I would have forgotten if you did not remind me :sweat_smile:

**Review by xbtmatt — APPROVED** [review]

**Pull request merged**